pkg/whitesource/sytemMock.go

```diff
@@ -0,0 +1,131 @@
+// +build !release
+
+package whitesource
```

Comment:
I would move the mocks into a dedicated `pkg/whitesource/mocks` package to separate them from the productive coding.

Reply:
What benefit do you see in that? We have the "*_test.go" files also right alongside the productive coding and I am not sure what additional changes this will necessitate.